### DISCUSSION TITLE
MQTT suddenly fails

### DISCUSSION THREAD

**Discussion by janbelder** [seaky/nspanel_pro_tools_apk#96] [Q&A]
After working for several days the MQTT fails to connect.
Anyone els with that problem and perhaps even how to solve it?
Removing and reinstalling the NSPanelPro tool did not solve the problem unfortunately....
Solved: It did not resolve the hostname(anymore?). Used IP address from Home assistant.....

**Comment by seaky** [comment]
I’ve been using my 5 devices with mqtt integration by HASS since 2024 feb and for me its rocksolid. 
I experienced some powerloss after long restart all device selfhealed itself I did not even do any intervention.

So I would check the mqtt server!

If you still experience problem, turn on debug and verbose log and post it here.

getting log
```
adb shell "ps -A | grep com.seaky.nspanelpro"

adb logcat --pid=<pid>

```

**Comment by seaky** [comment]
solved

**Comment by janbelder** [comment]
Hi Seaky, Although the problem is solved I want to thank you for the fast reply! And give you a thumbs up for this very useful tool! Wonder how you made it.... Jan On May 25, 2024, at 2:23 PM, seaky ***@***.***> wrote: solved — Reply to this email directly, view it on GitHub , or unsubscribe . You are receiving this because you authored the thread. Message ID: ***@***.***>
